jupyter_server/services/kernels/handlers.py

```diff
@@ -249,7 +360,7 @@ async def _register_session(self):
             await stale_handler.close()
         self._open_sessions[self.session_key] = self
 
-    def open(self, kernel_id):
+    async def open(self, kernel_id):
```

Comment:
This still has `async def open`, but it returns a Future. Needs to either be `async def` and `await future` *or* return Future, not both.

Reply:
Darn I carried this over. Fixed :)